tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
-    "outDir": "./dist",
```

Comment:
Compiled JS files are placed together with their TS source code to remain backward compatibility as it's possible to import files inside the package directly `import Block from "quill/formats/blot"`.